VueJS/index.js

```diff
@@ -4,10 +4,47 @@ import ListItem from './components/ListItem.js'
 new Vue({
   el: '#app',
   data: {
+    newTodoText: '',
+    counter:0,
+    editmode:[
+      false,false,false
```

Comment:
Try to put data as close together as possible. The whole point of object oriented programming is about putting things together that belong together, no?

So, please don't use a parallel array if you can just add the `editMode` to the todo objects.

But the ideal solution is not model the edit mode on the todo item. It's better of as some internal state of an `Item` component.